examples/embed/widget.py

```diff
@@ -103,7 +103,7 @@ def update_pyramid(self):
 pop = Population()
 pop.render()
 
-tag = embed.autoload_server(pop.layout)
+tag = embed.autoload_server(session_id=pop.session.id, pop.layout)
```

Comment:
I could add a little more info to some of the doc strings, will give that a shot.